README.md

```diff
@@ -2,61 +2,129 @@
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/fbf9f8e1bb7b4d2cbb1ca569014ed45b)](https://app.codacy.com/app/shehand/Go-social?utm_source=github.com&utm_medium=referral&utm_content=shehand/Go-social&utm_campaign=Badge_Grade_Dashboard) [![Build Status](https://travis-ci.org/shehand/Go-social.svg?branch=master)](https://travis-ci.org/shehand/Go-social) [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/scorelab/go-social)
 
-Today's community use mobile phones to make their life easier, and community-based mobile applications are famous among them. There are several kinds of community-based applications in use, and most of them relate to a specific domain. But core components have similar features. Addressing this issue and saving development time by introducing a common app template with customizable components for community-based mobile applications is the main purpose of this project.
+Go-social is a customizable React Native app template designed for community-based mobile applications. The template provides a common foundation for building such applications, saving development time and allowing developers to focus on adding unique features and functionality specific to their community.
 
-## Setup the project
+## Features
 
-> NOTE: If you have not yet installed react-native, use [this link](https://facebook.github.io/react-native/docs/getting-started) to install and configure react-native. **Make sure to use react-native cli rather than using expo cli**.
+This app template includes the following features:
 
-## Configure Go-social
+- Customizable components
+- Firebase Authentication and Real-time database
+- Feed with posts and comments
+- Social sharing and messaging functionality
+- Notifications for new posts and comments
+- Additional features such as easy theming will be implemented in the near future.
 
-- Clone the project
+## Getting Started
 
-  `git clone https://github.com/scorelab/Go-social.git`
+To get started, you'll need to have the following software installed on your machine:
 
-- Change the directory to the project folder
+- `Node.js`
+- `React Native CLI`
+- `VSCode` and `Android Studio` (for Android development)
+- `Xcode` (for iOS development)
 
-  `cd Go-social`
+Once you have these installed, follow these steps:
 
-- Install the node modules with the following command
+1. Clone the repository to your machine:
 
-  `npm install`
+```sh
+git clone https://github.com/scorelab/Go-social.git
+```
 
-### Configure the config.example.js file (IMPORTANT)
+2. Navigate to the project directory:
 
-> Open the project in any text editor you use. Open the `config.example.js` file in the `config` folder and follow the following steps.
+```sh
+npm install
+```
 
```

Comment:
I think this should be cd command?

Reply:
oh sorry for mistake actually i edited my last pr readme file. so there it both command was combined.